supernode/server/router.go

```diff
@@ -51,16 +52,20 @@ func initRoute(s *Server) *mux.Router {
 
 		// v1
 		// peer
-		{Method: http.MethodPost, Path: "/peers", HandlerFunc: s.registerPeer},
-		{Method: http.MethodDelete, Path: "/peers/{id}", HandlerFunc: s.deRegisterPeer},
-		{Method: http.MethodGet, Path: "/peers/{id}", HandlerFunc: s.getPeer},
-		{Method: http.MethodGet, Path: "/peers", HandlerFunc: s.listPeers},
+		{Method: http.MethodPost, Path: "/api/v1/peers", HandlerFunc: s.registerPeer},
```

Comment:
Could you please also update the swagger definition? THX.

Reply:
emmm,do u mean the comment?